src/components/table/examples/table-header-menu.tsx

```diff
@@ -0,0 +1,173 @@
+import { Component, h } from '@stencil/core';
+import { Column } from '../table.types';
+import { persons, Person } from './persons';
+
+/**
+ * Column header menu
+ *
+ * @link persons.ts
+ */
+@Component({
+    tag: 'limel-example-table-header-menu',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class TableExampleHeadermenu {
+    private tableData: Person[] = persons;
+    private columns: Column[] = [
+        {
+            title: 'Name',
+            field: 'name',
+            headerMenu: {
+                component: {
+                    name: 'limel-example-header-menu',
+                    icon: 'menu',
+                    items: [
+                        { text: 'All' },
+                        { text: 'Me' },
+                        { text: 'Contains' },
+                        { text: 'Equals' },
+                        { text: 'Not equals' },
+                    ],
+                },
+            },
+        },
+        {
+            title: 'Age',
+            field: 'age',
+            headerMenu: {
```

Comment:
Maybe we do not need a menu component for each column in this example. Since the menu items does not do anything at the moment, this whole example is mostly a big chunk of duplicated menu items that does not add much to the reader. What do you think?